src/courtroom.cpp

```diff
@@ -4708,6 +4715,31 @@ void Courtroom::announce_case(QString title, bool def, bool pro, bool jud,
   }
 }
 
+void Courtroom::start_clock()
+{
+  ui_clock->start();
+}
+
+void Courtroom::start_clock(qint64 msecs)
+{
+  ui_clock->start(static_cast<int>(msecs));
+}
+
+void Courtroom::set_clock(qint64 msecs)
+{
+  ui_clock->set(static_cast<int>(msecs), true);
+}
+
+void Courtroom::pause_clock()
```

Comment:
you didn't add this to the header

Reply:
I did tho https://github.com/AttorneyOnline/AO2-Client/pull/266/files#diff-84657662b5c61157314576af46e23adfR254

Reply:
wait wtf fuck I forgot to ctrl+S